backend/backend/MemoryGameWithNumbers/Services/MemoryGameService.cs

```diff
@@ -0,0 +1,50 @@
+namespace backend.MemoryGameWithNumbers.Services;
+
+public class MemoryGameService
+{
+    private List<int?>? _correctSequence;
```

Comment:
Imo cia nereikia `_correctSequence`, nes vistiek visi skaiciai bus in order, tai uztenka issaugot `maxNumber` reiksme.